tests/test_custom_image_host.cpp

```diff
@@ -337,14 +289,23 @@ TEST_F(CustomImageHost, info_for_unsupported_remote_throws)
 {
     mp::CustomVMImageHost host{"x86_64", &mock_url_downloader, default_ttl};
 
-    const std::string unsupported_remote{"snapcraft"};
+    const std::string unsupported_remote{"foo"};
     EXPECT_CALL(*mock_platform, is_remote_supported(unsupported_remote)).WillRepeatedly(Return(false));
 
     MP_EXPECT_THROW_THAT(host.info_for(make_query("xenial", unsupported_remote)), mp::UnsupportedRemoteException,
                          mpt::match_what(HasSubstr(fmt::format(
                              "Remote \'{}\' is not a supported remote for this platform.", unsupported_remote))));
 }
 
+TEST_F(CustomImageHost, info_for_full_hash_returns_empty_image_info)
+{
+    mp::CustomVMImageHost host{"x86_64", &mock_url_downloader, default_ttl};
+
+    auto info = host.info_for_full_hash("934d52e4251537ee3bd8c500f212ae4c34992447e7d40d94f00bc7c21f72ceb7");
```

Comment:
May adding const to `auto info`, auto catch things by value and non-const by default, since` info_for_full_hash` returns by value, then catching by value is fine; but const is something we have to add manually.

Reply:
Sure, makes sense and will do that.